aries_vcx/tests/pgwallet/wallet_plugin.rs

```diff
@@ -0,0 +1,92 @@
+use std::ffi::CString;
```

Comment:
Seems rather some kind of `libloading_utils.rs`? Again, I would suggest separating general and postgres-specific code.

Reply:
Changed to `wallet_plugin_loader`, the libloading code is very tailored for loading wallet plugins